data-structures/arrays/python/array_2D_output_all.py

```diff
@@ -0,0 +1,47 @@
+# Isaac Computer Science
+# Usage licensed under the Open Government Licence v3.0
+
+
+# Index values of each level
+LEVEL1 = 0 
+LEVEL2 = 1
+LEVEL3 = 2
+
+
+def create_2D_list():
+    """Create a two-dimensional list of words and levels"""
```

Comment:
Again, if you want to match the pseudocode you could loop initialise with empty strings and then set individual values